taskcluster/test/params/train.yml

```diff
@@ -0,0 +1,94 @@
+base_ref: origin/main
+base_repository: https://github.com/mozilla/firefox-translations-training
+base_rev: 6982b7bfc93e2054c5397aa14278a6c4fff63847
+build_date: 1693341713
+build_number: 1
+do_not_optimize: []
+enable_always_target: true
+existing_tasks: {}
+filters:
+- target_tasks_method
+head_ref: refs/heads/main
+head_repository: https://github.com/mozilla/firefox-translations-training
+head_rev: 6982b7bfc93e2054c5397aa14278a6c4fff63847
+head_tag: ''
+level: '3'
+moz_build_date: '20230829204153'
+next_version: null
+optimize_strategies: null
+optimize_target_tasks: true
+owner: example@users.noreply.github.com
+project: firefox-translations-training
+pushdate: 0
+pushlog_id: '0'
+repository_type: git
+target_tasks_method: train-target-tasks
+tasks_for: action
+training_config:
+  datasets:
+    devtest:
+    - flores_dev
+    - sacrebleu_wmt19
+    mono-src:
+    - news-crawl_news.2020
+    mono-trg:
+    - news-crawl_news.2020
+    test:
+    - flores_devtest
+    - sacrebleu_wmt20
+    train:
+    - opus_ada83/v1
+    - opus_GNOME/v1
+    - mtdata_Neulab-tedtalks_train-1-eng-rus
+  experiment:
+    backward-model: NOT-YET-SUPPORTED
+    best-model: chrf
+    bicleaner:
+      dataset-thresholds:
+        mtdata_Neulab-tedtalks_train-1-eng-rus: 0.6
+        opus_ada83/v1: 0.0
```

Comment:
So. This defaults to zero (not 0 dot 0) and taskgraph complaints that it got an integer instead of a float. I wonder if we can fix that... we were having the same issue triggering the action from the Taskcluster UI.

Reply:
I've noticed this as well. My guess is that it's a bug in taskgraph when it writes out parameters.

Reply:
I don't like it :( might dive into it a bit to see if I can fix it